bretm9/genrefy
# Nd generated playlist design

### What’s this PR do?  
- Allows a user to click on a saving and unsaving playlist icon to save and unsave a playlist.
- Formats some files for easier readability with longer lines.
- Adds the graphics needed for saving and unsaving playlist icon.
- Adds the toggleSaved method to trigger setState to re-render the saving graphic based on the isSaved flag inside playlist.
- Added basic styling to allow for clicking the star icon
 
### Where should the reviewer start?  
- Git pull this branch
- Working in App.tsx, PlaylistContainer.tsx, GeneratedPlaylist.tsx
 
### How should this be manually tested?  
- `npm start` and navigate to localhost:3000.
- Click on a genre to generated a new generated playlist.
- Click on the star icon beside the name of the generated playlist to change the graphic and set isSaved to true.
- Generate multiple playlist with other genres.
- Make sure that each playlist can be saved and unsaved individually.
- Navigate to saved page by click 'view saved' at top right.
- Ensure only saved playlist are showing in this screen.
- Click a playlist to be unsaved from saved page and ensure that playlist disappears from that screen.
 
### Any background context you want to provide?  
- Finalizing the design for the generated playlist will be done in the next branch.
 
### What are the relevant tickets?  
- closes #80
- closes #75
 
### Screenshots (if appropriate)  
- N/A
 
### Questions: 
- N/A
